### PR TITLE
Adding Documentation for required shop query parameter

### PR DESCRIPTION
**Pull request by jpotts18**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ You can configure the scope, which you pass in to the `provider` method via a `H
 
 * `scope`: A comma-separated list of permissions you want to request from the user. See [the Shopify API docs](http://docs.shopify.com/api/tutorials/oauth) for a full list of available permissions.
 
+* You **MUST** initiate the OmniAuth process by passing in a `shop` query parameter of the shop you’re requesting permissions for example. http://localhost:3000/auth/shopify?shop=example.myshopify.com.
```
Comment (dylanahsmith) [inline README.md]: I think this belongs in the Usage section

Comment (dylanahsmith) [inline README.md]: other than that, LGTM

**Comment by jpotts18** [comment]
It's yours now. Do what you want with it.

**Comment by dylanahsmith** [comment]
Manually merged as commit e738f0d1390c400c1b9311be94ede45b27d256a4 then edited with commit d6926463399166adc84ccbf582e27b62f299a0b1 to have this be part of the usage with a example form to illustrate how to use it.

Thanks for bringing these problems with the documentation to our attention

**Pull request closed without merging**